gradio/components/file_explorer.py

```diff
@@ -139,10 +139,8 @@ def postprocess(self, value: str | list[str] | None) -> FileExplorerData | None:
         return FileExplorerData(root=root)
 
     @server
-    def ls(self, value=None) -> list[dict[str, str]] | None:
+    def ls(self, _) -> list[dict[str, str]] | None:
```

Comment:
This change -- making the second argument requried -- is causing the tests to fail